api/alembic/versions/c92ba5b2b3d5_adds_audit_model.py

```diff
@@ -0,0 +1,41 @@
+"""Adds Audit model
+
+Revision ID: c92ba5b2b3d5
+Revises: 004ac48ffe18
+Create Date: 2021-04-12 14:33:00.507000
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'c92ba5b2b3d5'
+down_revision = '004ac48ffe18'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('audits',
+                    sa.Column('id', sa.Integer(), nullable=False),
+                    sa.Column('username', sa.String(), nullable=False),
```

Comment:
Of course.... it's not a create user.... hmmm.... maybe just "user" ? I don't know.

Reply:
I think you are correct in your first comment and my change was made with your second comment's understanding.

But a closer look at the document describes the `CREATE_USER` field as "The user or proxy account that created the record. **"
I should probably also change the username I'm persisting to this field as just "username" instead of the current "username@idir", but I'm not sure what format BCEID's are in.

Reply:
Oh no - don't touch the username - take it exactly as given. One doesn't even need idir for things to work. For example - I can create a username "sybrand" in Keylock, and configure things so that one could log in with keycloak, or with idir or with bceid. So just take the value you get and persist it blindly.